src/app/api/core/services/requestQueue.service.ts

```diff
@@ -0,0 +1,27 @@
+import { vercelConfig, zeploConfig } from '@/config'
+
+/**
+ * Wrapper around Zeplo which provides a queue service for requests
+ *
+ * https://zeplo.io/docs/getting-started/
+ */
+export class RequestQueueService {
+  /**
+   * Push a new request into the request queue
+   * @param url URL where request is to be made
+   * @param opts Provide optional request body or a trace id to identify this particular request
+   */
+  async push(url: string, opts?: { params?: object; traceId?: string }): Promise<Response> {
+    return await fetch(`https://zeplo.to/${zeploConfig.url}${url}`, {
```

Comment:
fine for now, but we might want to add constants for base url and urls, as there can be multiple calls.